### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by cquintana92**
This PR performs the following changes:

1. Upgrades some dependencies to the latest version.
2. Adapt the `webpack.config.js` to the upgrades. (mostly the `CopyWebpackPlugin`).
3. Upgrades the `.jshintrc` esversion (we were already using version 9 features and the IDE complained about it, so the only change it does is suppress these warnings).
4. Fix lint issues that arised due to (2).
5. Removed an unused and unmantained dependency (`vue-textarea-autosize`).

A dependency that could not be upgraded to the latest version is `webpack` (it has only been upgraded to v4), as some dependencies only support up to webpack 4, and crash if used with webpack v5.

There are still a few high severity vulns due to the aforementioned libs that only support webpack 4. They can be reviewed in the future.

```
λ npm audit
# npm audit report

glob-parent  <5.1.2
Severity: high
Regular expression denial of service in glob-parent - https://github.com/advisories/GHSA-ww39-953v-wcq6
fix available via `npm audit fix --force`
Will install webpack@5.72.1, which is a breaking change
node_modules/watchpack-chokidar2/node_modules/glob-parent
  chokidar  1.0.0-rc1 - 2.1.8
  Depends on vulnerable versions of glob-parent
  node_modules/watchpack-chokidar2/node_modules/chokidar
    watchpack-chokidar2  *
    Depends on vulnerable versions of chokidar
    node_modules/watchpack-chokidar2
      watchpack  1.7.2 - 1.7.5
      Depends on vulnerable versions of watchpack-chokidar2
      node_modules/watchpack
        webpack  4.44.0 - 4.46.0
        Depends on vulnerable versions of watchpack
        node_modules/webpack

5 high severity vulnerabilities
```

This PR closes:

* https://github.com/simple-login/browser-extension/pull/110
* https://github.com/simple-login/browser-extension/pull/111
* https://github.com/simple-login/browser-extension/pull/112
* https://github.com/simple-login/browser-extension/pull/115
* https://github.com/simple-login/browser-extension/pull/116
* https://github.com/simple-login/browser-extension/pull/120
* https://github.com/simple-login/browser-extension/pull/124
* https://github.com/simple-login/browser-extension/pull/127
* https://github.com/simple-login/browser-extension/pull/137

### CODE REVIEW

File: src/popup/popup.js [hunk 1]
```diff
@@ -75,7 +73,6 @@ async function initApp() {
   Vue.use(VModal, { dialog: true });
   Vue.use(VueRouter);
   Vue.use(ToggleButton);
-  Vue.use(TextareaAutosize);
```
Comment (nguyenkims) [inline src/popup/popup.js:78]: Do you know where do we use this one?

**Review by nguyenkims — APPROVED** [review]
LGTM, just left a question

File: src/popup/popup.js (hunk 1 above)
Comment (cquintana92) [reply src/popup/popup.js:78]: We don't use it anywhere!

If we look for it on `master`:

```
λ rg TextareaAutosize
src/popup/popup.js
13:import TextareaAutosize from "vue-textarea-autosize";
78:  Vue.use(TextareaAutosize);
```

Comment (nguyenkims) [reply src/popup/popup.js:78]: I tested and removing TextareaAutosize breaks the alias note editing. In the current version, the note automatically resizes when user enters
Screen Shot 2022-06-03 at 15 20 53

whereas in this version, the note doesn't even allow editing ... (it's hard to see from a screenshot)

image

Comment (cquintana92) [reply src/popup/popup.js:78]: Fixed it!

**Review by nguyenkims — APPROVED** [review]
LGTM!

**Pull request merged**